hack/gen-crd.sh

```diff
@@ -2,6 +2,15 @@
 
 set -eu
 
+function annotate_crd() {
```

Comment:
Is there a way to do this in go code rather than here?

Reply:
You mean a go program to invoke the original gen tool?

Reply:
Definitely, if there's a better way to do it, then we should create an issue for it. However, this already went in to master and all we're trying to do is backport it to 4.4 and 4.3.

Reply:
yeh there's OCPCLOUD-809 to follow up on this separately. We should probably use kustomize or similar for the long term.